lua/avante/suggestion.lua

```diff
@@ -85,6 +85,7 @@ function Suggestion:suggest()
       vim.schedule(function()
         local cursor_row, cursor_col = Utils.get_cursor_pos()
         if cursor_row ~= doc.position.row or cursor_col ~= doc.position.col then return end
+        full_response = full_response:gsub("^`*%w*\n(.-)\n`*$", "%1")
```

Comment:
```suggestion
        full_response = full_response:gsub("^```%w*\n(.-)\n```$", "%1")
```

Would such an accurate match be better? Because I would worry about the following normal content, such as multi-line strings in certain languages:

```
`firstline
secondline`
```

Reply:
I think the accurate match version is better! I'll fix it.